pkg/manager/impl/util/digests_test.go

```diff
@@ -46,8 +46,8 @@ var compiledTaskDigest = []byte{
 	0xa, 0x22, 0x80, 0xb1, 0x8, 0x44, 0x53, 0xf3, 0xca, 0x60, 0x4, 0xf7, 0x6f}
 
 var compiledWorkflowDigest = []byte{
-	0x1c, 0x66, 0x45, 0x89, 0x8f, 0x5, 0xa5, 0x4f, 0xf5, 0xba, 0x6f, 0xd9, 0xd, 0x70, 0xf6, 0x86, 0xf, 0x8e, 0x7e, 0x1b,
-	0x80, 0x1d, 0xb9, 0x59, 0xe, 0x3, 0x50, 0x4d, 0x64, 0xeb, 0x13, 0xa2}
+	0xeb, 0x66, 0x44, 0xe8, 0x1c, 0xa8, 0x51, 0x7d, 0x3f, 0x33, 0xf0, 0x77, 0x95, 0x24, 0x84, 0xc2, 0xbe, 0x79, 0xcd,
```

Comment:
This is due to introducing of `google.golang.org/protobuf` as transitive dependency.

Reply:
interesting, @katrogan will this affect all existing digests to be invalidated, should be fine, but we might cause some re-registrations?

Reply:
This is where I most concerned about as well. I can dig a bit further to see whether this can be avoided.

Reply:
oh interesting! this could be a problem because some folks will get a warning about the digest changing for a workflow with the same identifier, so they will have to change the version parameter to reregister - which is unfortunate

Reply:
I managed to revert this change by downgrading a few deps, in latest commit. Still we will need to face this thing later: https://github.com/lyft/flyteadmin/pull/121#discussion_r482317790